Client.xcodeproj/project.pbxproj

```diff
@@ -5742,11 +5742,11 @@
 				"${BUILT_PRODUCTS_DIR}/RNDeviceInfo/RNDeviceInfo.framework",
 				"${BUILT_PRODUCTS_DIR}/RNFS/RNFS.framework",
 				"${BUILT_PRODUCTS_DIR}/RNSqlite2/RNSqlite2.framework",
+				"${BUILT_PRODUCTS_DIR}/RNUserAgent/RNUserAgent.framework",
 				"${BUILT_PRODUCTS_DIR}/RNViewShot/RNViewShot.framework",
 				"${BUILT_PRODUCTS_DIR}/React/React.framework",
 				"${BUILT_PRODUCTS_DIR}/Realm/Realm.framework",
 				"${BUILT_PRODUCTS_DIR}/RealmSwift/RealmSwift.framework",
-				"${BUILT_PRODUCTS_DIR}/RxAtomic/RxAtomic.framework",
 				"${BUILT_PRODUCTS_DIR}/RxSwift/RxSwift.framework",
```

Comment:
I don't know why this is removed. I didn't remove it, at least on purpose :-)

@mahmoud-cliqz @khaled-cliqz Shall I bring it back?

Reply:
Changes in project file are supposed to be committed, please don't bring it back unless @mahmoud-cliqz thinks otherwise

Reply:
@giota-cliqz  does `RxAtomic` exist as a target in the pod project after you run `pos install`?

Reply:
Ok I did a quick test on master branch. `RxAtomic` was present in the `pbxproj` file and when I did `pod install`, it got removed. So I guess it is fine not to bring it back.

But can you explain why it gets removed?

Reply:
Then I think it should have been removed before but the one who changed the podfile might forgot to push it

Reply:
Ok thx! Makes sense